interop-testing/src/test/java/io/grpc/testing/integration/StressTestClientTest.java

```diff
@@ -132,27 +133,27 @@ public void gaugesShouldBeExported() throws Exception {
     MetricsServiceGrpc.MetricsServiceBlockingStub stub = MetricsServiceGrpc.newBlockingStub(ch);
 
     // Wait until gauges have been exported
-    Iterator<GaugeResponse> responseIt = stub.getAllGauges(EmptyMessage.getDefaultInstance());
-    while (!responseIt.hasNext()) {
-      LockSupport.parkNanos(TimeUnit.MILLISECONDS.toNanos(100));
-      responseIt = stub.getAllGauges(EmptyMessage.getDefaultInstance());
-    }
-
     Set<String> gaugeNames = newHashSet("/stress_test/server_0/channel_0/stub_0/qps",
         "/stress_test/server_0/channel_0/stub_1/qps");
 
-    while (responseIt.hasNext()) {
-      GaugeResponse response = responseIt.next();
-      String gaugeName = response.getName();
+    List<GaugeResponse> allGauges =
+        newArrayList(stub.getAllGauges(EmptyMessage.getDefaultInstance()));
```

Comment:
Google is moving away from newArrayList.  If you want a copy, use ImmutableList.copyOf().